ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/CCDRepaymentPlan.java

```diff
@@ -1,7 +1,8 @@
-package uk.gov.hmcts.cmc.ccd.domain.ccj;
+package uk.gov.hmcts.cmc.ccd.domain;
 
 import lombok.Builder;
 import lombok.Value;
+import uk.gov.hmcts.cmc.ccd.domain.ccj.CCDPaymentSchedule;
```

Comment:
We should move `CCDPaymentSchedule` out of `cci` package as well and then do the same with CMC model.